src/src/facilitator/facilitator.controller.ts

```diff
@@ -97,9 +97,10 @@ export class FacilitatorController {
 
 	@Post('/exportCsv')
 	@UseGuards(new AuthGuard())
+	@UsePipes(ValidationPipe)
 	async exportFileToCsv(
 		@Req() request: any,
-		@Body() body: any,
+		@Body() body: FilterFacilitatorDto,
```

Comment:
where is this dto code? already committed?

Reply:
yes